### ISSUE TITLE
Add requirements.txt

### ISSUE THREAD

**Issue by penrods** [KCAstromechs/powerscore_display#2]
A standard requirements.txt file would make it a little easier for people to get started with this.  The format is simple, just a list of the PIP libraries, either with or without a version.  (see an example [here](https://github.com/MycroftAI/mycroft-core/blob/dev/requirements.txt) )

After that, users can just run ```pip install -r requirements.txt``` and be ready to roll!

**Comment by jojojo8359** [comment]
Agreed. All that should need to be downloaded with pip would be requests and beautifulsoup4, both listed in my pull request's readme.

**Comment by penrods** [comment]
That's what inspired me to create this issue.  :)
